src/pages/workspaces/index.tsx

```diff
@@ -59,7 +67,14 @@ const Workspaces = () => {
             </FcBar>
             <MiniBdList>
               {focusBoardList.map((focusBoardItem) => (
-                <li key={focusBoardItem.id}>{focusBoardItem.title}</li>
+                <Link
+                  key={focusBoardItem.id}
+                  href={`/boards/${focusBoardItem.id}`}
```

Comment:
Want to use constant instead

Reply:
I made the change. Thanks for the reminder!